classes/components/forms/context/PKPContextForm.php

```diff
@@ -71,6 +72,19 @@ public function __construct($action, $locales, $baseUrl, $context)
                 'groupId' => 'identity',
                 'value' => $context ? $context->getData('acronym') : null,
             ]))
+            ->addField(new FieldText('contactName', [
+                'label' => __('manager.setup.principalContact') .' '. __('common.name'),
+                'isRequired' => true,
+                // 'groupId' => 'principal',
```

Comment:
Probably, these commented lines should be removed before the merge, right?

Reply:
removed .